src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -102,7 +103,7 @@ class AppsResource @Inject() (
       case _ =>
         Response
           .status(409)
-          .entity(Json.obj("message" -> s"An app with id [${app.id}] already exists.").toString)
+          .entity(Json.obj("message" -> s"An app with id [${app.id}] already exists.").toString())
```

Comment:
I personally agree. It was the only occurrence without parentheses, while all other have one. See here for discussion: http://stackoverflow.com/questions/8124055/scala-tostring-parenthesize-or-not